py/forecast.py

```diff
@@ -39,11 +41,18 @@ def __init__(self):
         self.lrmin = 985.0
         self.lrmax = 1200.0
         # S/N weights evaluated at this Fourier mode
-        self.kt_w_deg=7.0 # ~ 0.1 h/Mpc
-        self.kp_w_kms=0.001 # ~ 0.1 h/Mpc
+        self.kt_w_deg=2.4 # ~ 0.035 h/Mpc (k=0.05, mu=0.72)
```

Comment:
This is more of a curiosity, why was it decided to change these values?

Reply:
Depending on the science case that you are after you might want to use a different scale to optimize its SNR. It looks like the original value was optimized for small-scale clustering (P1D), while the new value is optimized for larger scales (BAO).